docs/technical-documentation/checking-coding-standards.md

```diff
@@ -0,0 +1,9 @@
+# Checking Coding Standards
+
+Islandora Drupal modules follow the [Drupal communities coding standards](https://www.drupal.org/docs/develop/standards/coding-standards). You can check your code against these standards using [PHP_Codesniffer](https://www.drupal.org/docs/8/modules/code-review-module/installing-coder-sniffer).
+
+If you are running the Vagrant, you can run `phpcs` within the Drupal installation directory (on the Vagrant, that is `/var/www/html/drupal`) as follows:
+
+`./vendor/bin/phpcs --standard=/var/www/html/drupal/vendor/drupal/coder/coder_sniffer/Drupal yourfile`, where `yourfile` is the path to the PHP file you want to check. If you specify a path to a directory instead of a single file, all files in that directory will be checked.
```

Comment:
The path to the Drupal standard can be relative. So, a slightly shorter version of the argument is `--standard=vendor/drupal/coder/coder_sniffer/Drupal` if running from `/var/www/html/drupal` as the example describes.

Reply:
I usually run it from the web directory, so it often looks like `../vendor/bin/phpcs --standard=../vendor/drupal/coder/coder_sniffer/Drupal modules/custom/mymodule/src/`

Reply:
Yeah, you need to be in the `web` folder for drush commands, so that's where I always do everything too, even if you don't have to be there for this particular command.

Reply:
Thanks, I will add these examples as well.